frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -227,12 +226,6 @@ export const RobotPage = () => {
                             toggleDialog={toggleSkipMissionDialog}
                         />
                         <StyledSmallContainers>
-                            {selectedRobot.model.type === RobotType.TaurobInspector && (
-                                <StyledSmallContainer>
-                                    <PressureTable />
-                                </StyledSmallContainer>
-                            )}
-
                             {selectedRobot.model.type === RobotType.TaurobInspector && (
                                 <StyledSmallContainer>
```

Comment:
Is it also possible to remove the smallContainers wrapper as there is only one left?